target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -488,6 +491,7 @@ ath79_setup_macs()
 		wan_mac=$(fritz_tffs -n macb -i $(find_mtd_part "tffs (1)"))
 		;;
 	compex,wpj344-16m|\
+	compex,wpj558-16m|\
```

Comment:
You set up two ethX in DTS, so please set the address there.

Since you only use one of these in 02_network, though, it's probably just your configuration being wrong and this may be kept eventually.